lib/rex/post/sql/ui/console/command_dispatcher/client.rb

```diff
@@ -134,22 +134,34 @@ def run_query(query)
               end
 
               def cmd_query(*args)
-                @@query_opts.parse(args) do |opt, idx, val|
+                show_help = false
+                call_interactive = false
+
+                @@query_opts.parse(args) do |opt, _idx, val|
                   case opt
+                  when nil
+                    show_help = true if val == 'help'
+                    break
                   when '-h', '--help'
-                    cmd_query_help
-                    return
+                    show_help = true
+                    break
                   when '-i', '--interact'
-                    cmd_query_interactive
-                    return
+                    call_interactive = true
+                    # break statements make whatever flag comes first take precedence; removing would make help take precedence. Is this the pattern we want? Should we change it? Or listen to both flags?
```

Comment:
Let's make sure comment is removed before we merge the PR
```suggestion
```

For context, among common CLI utilities both patterns are observed: some use whatever flag that is written first, some ignore everything and display help.